app/models/concerns/annexes/image_models.rb

```diff
@@ -8,4 +8,8 @@ module Annexes::ImageModels
 
     accepts_nested_attributes_for :image_models, allow_destroy: true, reject_if: :all_blank
   end
+
+  def organization_id
+    conclusion_review.review.organization_id
+  end
```

Comment:
Esto tiene un delegate, porque si lo ponés así se va a romper en el resto de los modelos lo que es un polimórfico. Debería funcionar, porque el conclusion_review tiene asociado directo un organization_id ¿Te tiró algún error?

Reply:
Claro, en ImageModel tiene un delegate de organization_id al belongs_to, en este caso seria un Annex. Por eso le tuve que agregar el metodo. Si fuese derecho al ConclusionReview estariamos bien, pero la imagen como pertenece al anexo se lo tuve que meter en el modelo de Annex. Si se lo saco me tira un error en ese metodo de organization_id.

Reply:
Ahh claro, me colgué con el nombre del concern y creí que eso lo habías puesto en el modelo ImageModel... jeje, perdón por eso.

Igual ahí deberías poder hacer lo siguiente:
```suggestion
  def organization_id
    conclusion_review.organization_id
  end
```
Porque aunque no recuerdo si está declarada la relación _explícita_, si está seguro la columna en la base de datos.

Reply:
funciono perfecto. Genial